### PR TITLE
[ZEPPELIN-6048] Write a Dockerfile for jdbc interpreter image build

### PR DESCRIPTION
**Pull request by seung-00**
### What is this PR for?
This PR adds a Dockerfile to build a JDBC interpreter container. It’s the second interpreter to be dockerized, after the shell interpreter (#4781). Like I said in the previous PR, the configuration and testing are a little complex at the moment. I plan to improve this in future work.

### What type of PR is it?
Feature
### What is the Jira issue?
* [ZEPPELIN-6048](https://issues.apache.org/jira/browse/ZEPPELIN-6048)
### How should this be tested?
* Build and run the docker container
* Verify that the jdbc interpreter container communicates with the zeppelin server

### Questions:
* Does the license files need to update? Yes
* Is there breaking changes for older versions? No
* Does this needs documentation? Yes. Docs added

### CODE REVIEW

File: shell/Dockerfile [hunk 1]
```diff
@@ -40,4 +40,11 @@ ENV SHELL_INTERPRETER_PORT=8081
 
 RUN chmod +x ./bin/interpreter.sh
 
-CMD ./bin/interpreter.sh -d interpreter/sh -c host.docker.internal -p ${INTERPRETER_EVENT_SERVER_PORT} -r ${SHELL_INTERPRETER_PORT}:${SHELL_INTERPRETER_PORT} -i sh-shared_process -l ./local-repo -g sh
+CMD ["./bin/interpreter.sh", \
+    "-d", "interpreter/sh", \
```
Comment (jongyoul) [inline shell/Dockerfile:44]:
```suggestion
    "-d", "interpreter/jdbc", \
```
??

Comment (seung-00) [reply shell/Dockerfile:44]: @jongyoul 
This is a Dockerfile for a shell interpreter, not JDBC. It seems I caused confusion by reformatting the shell Dockerfile this PR

Comment (seung-00) [reply shell/Dockerfile:44]: @jongyoul ping

Comment (jongyoul) [reply shell/Dockerfile:44]: Sorry I missed it.

**Review by jongyoul — APPROVED** [review]

**Pull request merged**